Index_v4

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<link rel="stylesheet" href="version3.css">
+<link rel="stylesheet" href="version4.css">
```

Comment:
I had to change the href from version 3 to version 4, as version 4 had my css for my footer layout. I am using css for my footer layout. href is the Hypertext REFerence which is used in HTML code to create links to other pages.